app/src/main/java/com/sopt/now/compose/HomeScreen.kt

```diff
@@ -13,7 +13,7 @@ import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.unit.dp
 
 @Composable
-fun HomeUi(viewModel: HomeViewModel) {
+fun HomeScreen(viewModel: HomeViewModel) {
```

Comment:
```suggestion
fun HomeScreen(viewModel: HomeViewModel = viewModel()) {
```

Compose에서 ViewModel은 함수 내에서 멤버변수로 직접 사용하거나 매개변수로 주입받는 경우에는 이런식으로 디폴트 매개변수로 주입받아서 호출부에서는

```kotlin
HomeScreen()
```

으로만 사용할 수 있게 작성하는 경향이 있습니다. 이 역시 취향의 영역이니 다른 분들 코드 종합적으로 보고, 직접 코드를 적어보시면서 어떤게 더 좋을지 판단해보시는 것이 좋을 것 같아요.